src/popup/popup.js

```diff
@@ -4,6 +4,28 @@ const EXISTING_PATH_SELECTOR_ID = "existing-path-selector"
 const NEW_PATH_INPUT_ID = "new-path-input"
 const PATH_SELECTOR_IDS = [EXISTING_PATH_SELECTOR_ID, NEW_PATH_INPUT_ID]
 
+chrome.runtime.onMessage.addListener(function(request) {
+    if (request.action == "getTitle") {
+      message.innerHTML = request.source;
+    }
+  });
+
+  function onWindowLoad() {
+
+    var message = document.getElementById('message');
+  
+    chrome.tabs.executeScript(null, {
+      file: "content.js"
```

Comment:
is this the only way to inject data into popup?
if so, let's name the script something else. Something more specific and descriptive.

Reply:
Let's rename `content.js` to something like `metadata-fetcher.js` or something similar